package.json

```diff
@@ -1,124 +1,125 @@
 {
```

Comment:
What did you change here?

Reply:
If you check package-lock.json you will see the real change. I added "mini-css-extract-plugin": "^0.8.0",